ISISNeutronMuon/HLM_View
# Ticket9 fix gitignore and logging

## Description of work done
Updated gitignore structure so that the `logs/` folder is tracked but its content is not.
Removed `logs/` folders contents.

## Fixes
#9 

## To test
Run the view and generate some logs. 
Check that the files are properly generated.
Check that git does not track them.
